roles/models.py

```diff
@@ -10,6 +10,12 @@
 from roles.roles import Staff, Instructor
 
 
+NON_LEARNERS = [
```

Comment:
please add a comment before this to say that this list needs to be updated in case we add new roles